integration-tests/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## `web-tests`
 
-The `web-tests` module contains a simplistic Spine application.
+The `web-tests` module contains a simplified Spine application.
```

Comment:
It is not simplified. It's just as simple as it gets. Maybe,  just `simple` is more suitable.